src/lib/offline-api/validator/errors.ts

```diff
@@ -64,6 +64,21 @@ const errors = {
       INVALID_VALIDATION_PARAMETER: (propName, expectedType, actualType) => {
         return `"${propName}" validation expected to be "${expectedType}", but got "${actualType}"`
       }
+    },
+    initialValue: {
+      TYPE_MISMATCH: (fieldId, valueType, locale, fieldType) => {
```

Comment:
Should we consider validations here as well? Like when you allow a list of values or numbers within a range and create some sort of `VALIDATED_TYPE_MISMATCH` or is this too much effort to justify its usefulness?

Reply:
We currently allow saving a content type with invalid initial field values 🙂

Reply:
Oh okay, so we do rely on the entry validations later on 💡 . 

> Nothing to see here, keep walking folks..